packages/webapp/src/inductions/components/induction-step-endorsement.tsx

```diff
@@ -297,46 +319,67 @@ const EndorsingForm = ({
 };
 
 interface DonationFormProps {
+    isCommunityActive?: boolean;
     isInvitee: boolean;
     isLoading: boolean;
     setReviewStep: (step: "profile" | "video") => void;
     submitDonation: () => void;
 }
 const DonationForm = ({
+    isCommunityActive,
     isInvitee,
     isLoading,
     setReviewStep,
     submitDonation,
 }: DonationFormProps) => {
+    const [isProfileReviewed, setReviewedProfile] = useState(false);
     return isInvitee ? (
         <div className="space-y-3">
             <Text>
-                This is your last chance to review your profile for completeness
-                and accuracy. If anything needs to be corrected,{" "}
+                This is your last chance to review your profile below for
+                completeness and accuracy. If anything needs to be corrected,{" "}
                 <Link onClick={() => setReviewStep("profile")}>click here</Link>
-                . Keep in mind that modifying your profile will require your
-                endorsers to submit their endorsements again.
+                .
+                {isCommunityActive &&
```

Comment:
When the time comes (soon) to refactor some of this...all these ternary checks and branching render logic indicates to me we should just make separate components for many of these views that have inviter/invitee/endorser/isCommunityActive roles...even if some of the text is shared among them.

Reply:
yes! agreed.